src/rocker/templates/nvidia_snippet.Dockerfile.em

```diff
@@ -25,4 +25,5 @@ COPY --from=glvnd /usr/share/glvnd/egl_vendor.d/10_nvidia.json /usr/share/glvnd/
 
 
 ENV NVIDIA_VISIBLE_DEVICES ${NVIDIA_VISIBLE_DEVICES:-all}
-ENV NVIDIA_DRIVER_CAPABILITIES ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
+ENV NVIDIA_DRIVER_CAPABILITIES ${NVIDIA_DRIVER_CAPABILITIES:-compute,utility}
+ENV NVIDIA_DRIVER_CAPABILITIES ${NVIDIA_DRIVER_CAPABILITIES},graphics
```

Comment:
Thanks for the pointers to the default modes. That looks good to add. Does will this replace #126 for your use cases?

However I'd like to continue to support the ability for the user to manually specify the arguments and then add `,graphics` 

```suggestion
ENV NVIDIA_DRIVER_CAPABILITIES ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
```

Reply:
@tfoote Thank you for your comment!

> Does will this replace https://github.com/osrf/rocker/pull/126 for your use cases?

Yes it does because we install CUDA libraries in our image.
See the following links for details.
- https://github.com/autowarefoundation/autoware/blob/ba9d3c4e36d58cbdbd29b85d4b4f54fde001fa0c/docker/autoware-universe/Dockerfile#L29
- https://github.com/autowarefoundation/autoware/blob/ba9d3c4e36d58cbdbd29b85d4b4f54fde001fa0c/setup-dev-env.sh#L126
- https://github.com/autowarefoundation/autoware/blob/ba9d3c4e36d58cbdbd29b85d4b4f54fde001fa0c/ansible/roles/cuda/tasks/main.yaml

> However I'd like to continue to support the ability for the user to manually specify the arguments and then add ,graphics

Sure. I'll commit your suggestion.